docs/vocabulary.md

```diff
@@ -8,27 +8,117 @@ hide_title: true
-
-Coming Soon:
-
-* Respiratory Condition: ICD-10-CM J00-J99
-* Invasive mechanical ventilation: Any intubation procedure or mechanical ventilation-associated procedure, excluding supplemental oxygen (high or low flow), CPAP, and BiPAP, ECMO (SNOMED CT, ICD-10-CM).
+Value sets supporting cohort definitions. Listing of value sets is in alphabetical order. Some value sets refer to an outside source or repository, and others are expanded here.
+
+### BCG Vaccine Products for Intravesical Use​ VS
+​
+Scope/Purpose: Identifies BCG vaccine products for intravesical use
+
+Inclusions: Live BCG for intravesical use
+
+Exclusions: None
+
+| Code | Description | Code System | Version | Status |
+|-----|------|------|------|-----|
+| 90586 | Bacillus Calmette-Guerin vaccine (BCG) for bladder cancer, live, for intravesical use​​ | HCPCS​ | 2019AB | Active
+| ​19 | Bacillus Calmette-Guerin vaccine | CVX | ​ | Active
+
+
+### Confirmed COVID-19 Infection VS
+
+Location: VSAC
+
+OID: [2.16.840.1.113762.1.4.1032.117](https://vsac.nlm.nih.gov/valueset/2.16.840.1.113762.1.4.1032.117/expansion/20200410)
+
+Code Systems: ICD10CM, SNOMEDCT
+
+Type: Grouping
+
+### Convalescent Serum of COVID-19 Products VS
+
+TBD
+
+### Hydroxychloroquine VS
+
+TBD
+
+### Intravesical Administration of BCG Procedure VS
+
+Scope/Purpose: Identifies procedures for intravesical administration of live BCG vaccine. Used to identify patients who received BCG as cellular therapy for bladder cancer.
+
+Inclusions: Procedures specific to the administration of live BCG into the bladder​ (active and inactive terms)
+
+Exclusions: Procedures not specific to BCG​ AND Administration of BCG other than in the intravesical space
+
+| Code | Description | Code System | Version | Status |
+|-----|------|------|------|-----|
+| J9031 | Bcg (intravesical) per instillation​ | HCPCS​ | 2020 | Inactive |
+| J9030​ | Bcg live intravesical instillation, 1 mg​ | HCPCS | 2020​ | Active |
+​301879009​ | Instillation of bacillus Calmette-Guerin into the bladder (procedure)​ | SNOMED CT | 202003​ | Active |
+
+### Intravesical Administration of Therapeutic Agent Procedure VS
+
+Scope/Purpose: Identifies procedures indicative of intravesical administration of therapeutic agent.
+
+Inclusions: Procedures representing the administration of unspecified therapeutic agent into bladder
+
+Exclusions: Procedures specific to BCG (covered by [Intravesical Administration of BCG Procedure VS](#intravesical-administration-of-bcg-procedure-vs))
+
+| Code | Description | Code System | Version | Status |
+|-----|------|------|------|-----|
+| 51720 | Bladder instillation of anticarcinogenic agent (including retention time)​ | CPT​ | 2019AB | Active
+| ​44194005 | Urinary bladder instillation of anticarcinogenic agent (procedure)​ | SNOMED CT | 202003​ | Active
+| ​712753009 | Administration of chemotherapeutic agent into bladder via intravesical route (procedure) | SNOMED CT | 202003​ | Active
+
+### Intravesical Route VS
```

Comment:
I don't think we use this in our logic, do we? The procedures we're looking for have a precoordinated route, I believe

Reply:
The lastest powerpoint was slightly messed up. It says: 

![image](https://user-images.githubusercontent.com/492232/83884695-4447e100-a713-11ea-9f53-34ba80fdb3be.png)

which says there are two procedure codes. It should be "Procedure code in Intravesical Adminstration of 

![image](https://user-images.githubusercontent.com/492232/83884718-5164d000-a713-11ea-87d6-3987d97c1d67.png)

Reply:
I'm open to suggestion and could remove this if you think it is not happening "in the wild"